core/controllers/base_test.py

```diff
@@ -1876,6 +1922,17 @@ def test_cannot_access_exploration_with_missing_schema(self):
             self.assertEqual(response['error'], error_msg)
         self.logout()
 
+    def test_can_access_exploration_with_default_value_in_schema(self):
+        self.login(self.OWNER_EMAIL)
+
+        with self.swap(self, 'testapp', self.mock_testapp3):
+            response = self.get_json('/mock_play_exploration')
```

Comment:
Please add a check to confirm that the default value is being updated.

Reply:
Added if condition in get and put methods of mockhandlers, PTAL.
Thank you.